Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -210,6 +210,7 @@ protected virtual void OnElementChanged(ElementChangedEventArgs<Label> e)
 			if (e.OldElement != null)
 			{
 				e.OldElement.PropertyChanged -= OnElementPropertyChanged;
+				this.MaybeRequestLayout();
```

Comment:
@paymicro does this only make sense here inside the LabelRenderer? Or is this something that needs to be applied more generally for the other fast renderers as well?  ButtonRenderer is probably taken care of because of layout manager? but what about the ImageRenderer?

Reply:
@PureWeen yes. This was for `LabelRenderer` only. 
I tested Button and Image fast renderer's and result is fine

| Button | Image |
|:------:|:-----:|
|   ![Screenshot_1](https://user-images.githubusercontent.com/27482193/60895002-7e430580-a26c-11e9-8aae-864d3f329737.png) |  ![Screenshot_2](https://user-images.githubusercontent.com/27482193/60895003-7e430580-a26c-11e9-974f-55c42c49511d.png) |